src/util.cpp

```diff
@@ -1423,11 +1423,14 @@ bool CLocale::IsCountryCodeSupported ( unsigned short iCountryCode )
 #if QT_VERSION >= QT_VERSION_CHECK( 6, 0, 0 )
     // On newer Qt versions there might be codes which do not have a Qt5 equivalent.
     // We have no way to support those sanely right now.
+    if ( iCountryCode >= qt6CountryToWireFormatLen )
```

Comment:
I think the briefer
```
return ( iCountryCode < qt6CountryToWireFormatLen ) &&
    qt6CountryToWireFormat[iCountryCode] != -1;
```
might be easier to read.  Even so, it could still do with a comment saying what `qt6CountryToWireFormatLen` is (though it's fairly obvious).

Reply:
Added a comment (didn't collapse it to a single boolean expression though; if you want to, feel free to ;)).

Reply:
OK, I'm happy enough with the comment in place.  It makes it clear that, perhaps, in future the check could be removed.